chromadb/server/fastapi/__init__.py

```diff
@@ -195,20 +195,18 @@ def delete_collection(self, collection_name: str) -> None:
 
     def add(self, collection_id: str, add: AddEmbedding) -> None:
         try:
-            result = self._api._add(
+            self._api._add(
                 collection_id=_uuid(collection_id),
-                embeddings=add.embeddings,
+                embeddings=add.embeddings,  # type: ignore
```

Comment:
Added to quiet this error:
`chromadb/server/fastapi/__init__.py:221: error: Argument "embeddings" to "_upsert" of "API" has incompatible type "Optional[List[Any]]"; expected "List[List[Union[int, float]]]"  [arg-type]`

It looks like this is expected, given this comment in fastapi/types?
```
    # Pydantic doesn't handle Union types cleanly like Embeddings which has
    # Union[int, float] so we use Any here to ensure data is parsed
    # to its original type.
 ```